### DISCUSSION TITLE
Does upgrading Postgres from the dashboard also upgrade PostgREST?

### DISCUSSION THREAD

**Discussion by eensander** [supabase/supabase#23457] [Questions] [answered]
Hello,

I was wondering, does the option "Upgrade project" on the infrastructure tab of the project overview also upgrade the version of **PostgREST**, next to upgrading Postgres?

Thanks in advance.

**Comment by GaryAustin1** [comment, marked as answer]
Yes.

**Comment by jdx** [comment]
This was fixed in v2026.1.8 via #7830. The fix validates lockfile URLs against expected asset patterns and refreshes when there's a mismatch (e.g., when the aqua registry format changed from raw binary to tar.gz).

You can update your workflow to use the latest version:

```yaml
- uses: jdx/mise-action@v3
  # v2026.1.8+ includes the fix
```

Or explicitly pin to 2026.1.8:

```yaml
- uses: jdx/mise-action@v3
  with:
    version: 2026.1.8
```
